packages/backend/test/git.test.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import simpleGit from 'simple-git';
+import os from 'os';
+import uniqueDirectoryName from 'short-uuid';
+import { cloneGitRepo } from '../src';
+
+jest.mock('fs');
+jest.mock('simple-git');
+jest.mock('short-uuid');
+jest.mock('os');
+jest.mock('os');
```

Comment:
Probably
```suggestion
jest.mock('os');
```
is enough 🙂